### PR TITLE
LP-2702 change application view/assess process for admin

### PR DESCRIPTION
**Pull request by annas-usman**
[Ticket](https://philanthropyu.atlassian.net/browse/LP-2702)

### CODE REVIEW

**Review by ahmed-faraz46 — APPROVED** [review]
LGTM 👍 But please have a look if we need to make any changes in the existing unit tests or to add any new unit tests

**Comment by omnipreneurshipA** [comment]
## Diff Coverage Report - ADG
Check-out complete **[report](https://app.circleci.com/pipelines/github/OmnipreneurshipAcademy/edx-platform/1911/workflows/workflows_id/jobs/1911/artifacts)**. Coverage !!! :rocket:
```
-------------
Diff Coverage
Diff: origin/develop...HEAD, staged and unstaged changes
-------------
openedx/adg/lms/applications/admin.py (100%)
openedx/adg/lms/applications/models.py (100%)
openedx/adg/lms/applications/tests/test_admin/test_user_application_adg_admin.py (100%)
openedx/adg/lms/applications/tests/test_models.py (100%)
-------------
Total:   6 lines
Missing: 0 lines
Coverage: 100%
-------------
```

File: openedx/adg/lms/applications/tests/test_models.py [hunk 1]
```diff
@@ -186,14 +186,14 @@ def test_work_experience_string_representation(work_experience):
 @pytest.mark.django_db
 def test_submitted_applications_manager():
     """
-    Test that the SubmittedApplicationsManager returns only submitted applications.
+    Test that the SubmittedApplicationsManager returns only applications with submitted written applications.
```
Comment (harris-irfan) [inline openedx/adg/lms/applications/tests/test_models.py:189]: same as above

*returns only submitted written applications

**Comment by omnipreneurshipA** [comment]
## Diff Coverage Report - ADG
Check-out complete **[report](https://app.circleci.com/pipelines/github/OmnipreneurshipAcademy/edx-platform/1914/workflows/workflows_id/jobs/1914/artifacts)**. Coverage !!! :rocket:
```
-------------
Diff Coverage
Diff: origin/develop...HEAD, staged and unstaged changes
-------------
openedx/adg/lms/applications/admin.py (100%)
openedx/adg/lms/applications/models.py (100%)
openedx/adg/lms/applications/tests/test_admin/test_user_application_adg_admin.py (100%)
openedx/adg/lms/applications/tests/test_models.py (100%)
-------------
Total:   6 lines
Missing: 0 lines
Coverage: 100%
-------------
```

File: openedx/adg/lms/applications/models.py [hunk 2]
```diff
@@ -48,9 +48,11 @@ def set_is_prerequisite_courses_passed(self):
 
     def set_is_written_application_completed(self):
```
Comment (harris-irfan) [inline openedx/adg/lms/applications/models.py:49]: Maybe update the method name to reflect the changes in method.

`set_is_written_application_completed_with_date` is an option.

File: openedx/adg/lms/applications/models.py [hunk 3]
```diff
@@ -48,9 +48,11 @@ def set_is_prerequisite_courses_passed(self):
...
         """
-        Mark written_application objective as complete i.e set is_written_application_completed to True.
+        Mark written_application objective as complete i.e set is_written_application_completed to True and add current
+        date as submission_date.
         """
         self.is_written_application_completed = True
+        self.submission_date = date.today()
```
Comment (harris-irfan) [inline openedx/adg/lms/applications/models.py:55]: This submission date is also being set in the ApplicationHub method `submit_application_for_current_date`. So according to the current implementation, the date we set here would be overridden when the application hub is submitted. I know that as per discussion, the application hub logic is to be updated, but the developer might miss out this case. So either you can change the `submit_application_for_current_date` method to not set this date there, or maybe keep a note of this, and add it to the application hub integration story.

**Review by harris-irfan — CHANGES_REQUESTED** [review]

File: openedx/adg/lms/applications/models.py (hunk 3 above)
Comment (annas-usman) [reply openedx/adg/lms/applications/models.py:55]: I have noted it, will add it in the integration story

**Comment by omnipreneurshipA** [comment]
## Diff Coverage Report - ADG
Check-out complete **[report](https://app.circleci.com/pipelines/github/OmnipreneurshipAcademy/edx-platform/1918/workflows/workflows_id/jobs/1918/artifacts)**. Coverage !!! :rocket:
```
-------------
Diff Coverage
Diff: origin/develop...HEAD, staged and unstaged changes
-------------
openedx/adg/lms/applications/admin.py (100%)
openedx/adg/lms/applications/models.py (100%)
openedx/adg/lms/applications/tests/test_admin/test_user_application_adg_admin.py (100%)
openedx/adg/lms/applications/tests/test_models.py (100%)
openedx/adg/lms/applications/tests/test_views.py (100%)
openedx/adg/lms/applications/views.py (100%)
-------------
Total:   13 lines
Missing: 0 lines
Coverage: 100%
-------------
```

**Review by harris-irfan — APPROVED** [review]
LGTM 👍

**Comment by omnipreneurshipA** [comment]
## Diff Coverage Report - ADG
Check-out complete **[report](https://app.circleci.com/pipelines/github/OmnipreneurshipAcademy/edx-platform/1923/workflows/workflows_id/jobs/1923/artifacts)**. Coverage !!! :rocket:
```
-------------
Diff Coverage
Diff: origin/develop...HEAD, staged and unstaged changes
-------------
openedx/adg/lms/applications/admin.py (100%)
openedx/adg/lms/applications/models.py (100%)
openedx/adg/lms/applications/tests/test_admin/test_user_application_adg_admin.py (100%)
openedx/adg/lms/applications/tests/test_models.py (100%)
openedx/adg/lms/applications/tests/test_views.py (100%)
openedx/adg/lms/applications/views.py (100%)
-------------
Total:   14 lines
Missing: 0 lines
Coverage: 100%
-------------
```

**Review by MunamMubashir-arbisoft — APPROVED** [review]
Great work! This made the theme code much cleaner

**Review by ahmed-faraz46 — APPROVED** [review]

**Pull request merged**